sublime_jedi/tooltips/markdown.py

```diff
@@ -53,17 +53,20 @@ def _prepare_signature(self, docstring):
         """
         pattern = (
             '(?x)'
-            '^([\w\. \t]+\.[ \t]*)?'        # path
-            '(\w+)'                         # function / object
-            '[ \t]*(\([^\)]*\))'            # arguments
-            '(\s*->\s*.*$)?'                # annotation (rest of line)
+            '^([\w\. \t]+\.[ \t]*)?'    # path
+            '(\w+)'                     # function / object
+            '[ \t]*(\([^\)]*\))'        # arguments
+            '(?:'
+            '(\s*->\s*.*?)'             # annotation: -> Type
+            '(--|$)'                    # inline comment: -- blabla
+            ')?'
```

Comment:
I typed in python shell:
```
>>> help(list.extend)

Help on method_descriptor:

extend(...)
    L.extend(iterable) -- extend list by appending elements from the iterable
```

I think it's just a plan text, and it should be as it is. 
And `-- blabla bla` is not an inline comments